Gymphony/src/app/app.module.ts

```diff
@@ -0,0 +1,38 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { provideAnimationsAsync } from '@angular/platform-browser/animations/async';
+import { AuthModule } from './features/auth/auth.module';
+import { SharedModule } from './shared/shared.module';
+import { HomeModule } from './features/home/home.module';
+import { NavbarComponent } from './core/components/navbar/navbar.component';
+import { ProfileIconComponent } from './core/components/profile-icon/profile-icon.component';
+import { AuthInterceptor } from './core/interceptors/auth.interceptor';
```

Comment:
Let's structure our imports like that
Divide them into 2 groups
Group 1: angular imports/libraries/third-party
empty line
Group 2: imports from your own files